tests/main.rs

```diff
@@ -144,14 +144,12 @@ fn date_from_timestamp_extremes() {
         Ok(dt) => panic!("unexpectedly valid, {dt}"),
         Err(e) => assert_eq!(e, ParseError::DateTooLarge),
     }
-    match Date::from_timestamp(-30_610_224_000_000, false) {
+    let d = Date::from_timestamp(-62_167_219_200_000, false).unwrap();
+    assert_eq!(d.to_string(), "0000-01-01");
+    match Date::from_timestamp(-62_167_219_200_001, false) {
         Ok(dt) => panic!("unexpectedly valid, {dt}"),
         Err(e) => assert_eq!(e, ParseError::DateTooSmall),
     }
-    let d = Date::from_timestamp(-11_676_096_000 + 1000, false).unwrap();
-    assert_eq!(d.to_string(), "1600-01-01");
-    let d = Date::from_timestamp(-11_673_417_600, false).unwrap();
-    assert_eq!(d.to_string(), "1600-02-01");
```

Comment:
Can we restore these test cases? Not immediately clear to me why they were deleted.

Reply:
I removed them because this test case is called `date_from_timestamp_extremes` and 1600 is no longer "an extreme value" (and I added 0000-01-01 instead.) I'll move them to `datetime_from_timestamp_specific`.